v1/PagerDutyConfig.dhall

```diff
@@ -0,0 +1,43 @@
+let imports = ../imports.dhall
+
+let Kubernetes = imports.Kubernetes
+
+let HTTPConfig = ./HTTPConfig.dhall
+
+let PagerDutyConfigDetail =
```

Comment:
Just to keep the convention in the repo seperate it to different file PagerDutyConfigDetail.dhall